### PR TITLE
duck 4.6.5

### PR DESCRIPTION
**Pull request by dkocher**

### CODE REVIEW

File: Library/Formula/duck.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ class Duck < Formula
     sha1 "d67d55eabd017376079eaed8ded19bc4c28f83d0" => :mountain_lion
   end
 
-  depends_on :java => ["1.7", :build]
+  depends_on :java => [:build, "1.7"]
```
Comment (xu-cheng) [inline Library/Formula/duck.rb]: Why do you want to add this? It makes `"1.7"` tag meaningless.

Comment (xu-cheng) [inline Library/Formula/duck.rb]: If it works for all java version, you should use `depends_on :java => :build`

Comment (dkocher) [inline Library/Formula/duck.rb]: We want to make sure it is built with Java 1.7.

Comment (xu-cheng) [inline Library/Formula/duck.rb]: Then you should leave it is. `depends_on :java => ["1.7", :build]` will make it build with Java 1.7.

**Comment by dkocher** [comment]
Anything else that prevents accepting this pull request?

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `4.6.5.17000` and we prefer changes to be squashed to a single commit.

**Pull request closed without merging**